charts/hono/values.yaml

```diff
@@ -1663,12 +1709,24 @@ amqpMessagingNetworkExample:
     # The default value ("Eclipse IoT;Hono") includes all adapters that authenticate with their
     # demo certificate.
     adapterUids: "Eclipse IoT;Hono"
-    # livenessProbeInitialDelaySeconds contains the value to use for the "initialDelaySeconds"
+    # [DEPRECATED: use probes instead] livenessProbeInitialDelaySeconds contains the value to use for the "initialDelaySeconds"
     # configuration property of the component's liveness probe.
+    # The value of the top level "livenessProbeInitialDelaySeconds" property will be used if not set.
     livenessProbeInitialDelaySeconds: 300
-    # readinessProbeInitialDelaySeconds contains the value to use for the "initialDelaySeconds"
+    # [DEPRECATED: use probes instead] readinessProbeInitialDelaySeconds contains the value to use for the "initialDelaySeconds"
     # configuration property of the component's readiness probe.
+    # The value of the top level "readinessProbeInitialDelaySeconds" property will be used if not set.
     readinessProbeInitialDelaySeconds: 20
+    # probes allows you to overwrite the global probes config values for this specific component
+    probes:
+      livenessProbe:
+        httpGet:
+          path: "/healthz"
+        periodSeconds: 9
+      readinessProbe:
+        httpGet:
+          path: "/healthz"
+        periodSeconds: 5
```

Comment:
IMHO we should set the initialDelaySeconds in the probes here and leave the deprecated properties empty. That way we also provide an example of how to set the values in the desired way ...

Reply:
sure, i'll fix it. I will also adapt that the probe config gets the priority over `livenessProbeInitialDelaySeconds` and `readinessProbeInitialDelaySeconds` as it is now the other way around i think.

Reply:
Actually, IMHO it is correct that `livenessProbeInitialDelaySeconds` and `readinessProbeInitialDelaySeconds` get priority over the values configured in the `probes` properties. In the end, we want to make sure that users who set these (deprecated) values still get these values applied (instead of the default values that you put into the `probes`). Or am I mistaken?

Reply:
you are right, i reverted the commit.